controllers/App.class.php

```diff
@@ -21,6 +21,9 @@ public function actionLogin($params=null){
         if(isset($_SESSION["user"])){
             header("Location: /");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Location: /" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.youseeu.com:9000/coding_rules#rule_key=php%3AS1192)